ouroboros-consensus-protocol/src/ouroboros-consensus-protocol/Ouroboros/Consensus/Protocol/Ledger/HotKey.hs

```diff
@@ -129,16 +139,41 @@ data HotKey c m = HotKey {
       -- | Return 'KESInfo' of the signing key.
     , getInfo    :: m KESInfo
       -- | Return 'True' when the signing key is poisoned because it expired.
+    , getOCertMaybe :: m (Maybe (OCert.OCert c))
     , isPoisoned :: m Bool
```

Comment:
```suggestion
      -- | Some haddock here
    , getOCertMaybe :: m (Maybe (OCert.OCert c))
      -- | Return 'True' when the signing key is poisoned because it expired.
    , isPoisoned :: m Bool
```

Reply:
Note that "because it expired" is wrong - a key can be in a poisoned state for a variety of reasons, e.g. because no key has ever been set for this 'HotKey', or because it has been explicitly erased.

Reply:
I was just pointing to the fact that that the haddocks were mixed up, you introduced `getOCertMaybe` in between `isPoisoned` and its haddock.